DigitalLearningSolutions.Web/DigitalLearningSolutions.Web.csproj

```diff
@@ -15,6 +15,10 @@
         <UserSecretsId>7ea176d2-09f5-4e3a-a3fa-7f3229882b70</UserSecretsId>
     </PropertyGroup>
 
+    <ItemGroup>
+      <AdditionalFiles Remove="C:\Users\Sherif.Olaboye\Source\Repos\DLSV2\StyleCop.json" />

```

Comment:
I'm not sure why this has  been added. It seems to reference a path on your local machine.

Reply:
It was added because I need to remove StyleCop.json file from the project

Reply:
Why did you need to remove StyleCop.json?

Reply:
I needed to remove the file to enable me build and run the project, cause I needed to asked about the file which I was told is not needed in the project